elementary/monitor/debug.py

```diff
@@ -17,6 +17,14 @@ def run(self) -> bool:
             self.config.profile_target,
         )
 
+        try:
+            dbt_runner.debug()
```

Comment:
This is problematic because `dbt debug` requires `git` which is not necessarily required.

Reply:
Do we know what is happening if we don't have git?
Does it still test the connection? If so, maybe we can extract only that part from the debug result?